src/style.css

```diff
@@ -0,0 +1,81 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  flex-grow: 1;
+  flex-basis: 300px;
+  color: #fff;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 100px;
+  background-color: red;
+}
+
+/*#region Background*/
+.block-2 {
+  background-color: rgb(80%, 0%, 0%);
+}
+
+.block-3 {
+  background-color: rgb(60%, 0%, 0%);
+}
+
+.block-4 {
+  background-color: rgb(40%, 0%, 0%);
+}
+
+.block-5 {
+  background-color: rgb(20%, 0%, 0%);
+}
+
+.block-6 {
+  background-color: black;
+}
+/*#endregion*/
+
+/*#region mediaWidth*/
+@media (min-width:600px) and (max-width:750px) {
+  .block-3 {
+    order: 1;
+  }
+  .block-5 {
+    order: 2;
+  }
+  .block-6 {
+    order: 3;
+  }
+
+}
+
+@media (min-width:900px) and (max-width:1200px) {
```

Comment:
that's what I see on widths > 1200
![image](https://user-images.githubusercontent.com/37161315/167660898-5cb037ef-0c7f-4dac-bf41-54ba591e1157.png)
remove this max-width part too